accounting/bank/setup/foreign_currency.rst

```diff
@@ -34,10 +34,10 @@ Configure currencies
 --------------------
 
 Once the Odoo is configured to support multiple currencies, you should
-activate the currencies you plan to work with. To do that, go the menu
+activate the currencies you plan to work with. To do that, go to the menu
 :menuselection:`Configuration --> Currencies`. All the currencies are created by default,
-but you should activate the ones you plan to support. (to activate a
-currency, check his active field)
+but you should activate the ones you plan to support (to activate a
+currency, select the corresponding field).
```

Comment:
Well... "active" is the name of the field/switch to be used:
![image](https://user-images.githubusercontent.com/7300760/63952158-58252f00-ca7f-11e9-9e86-20cfc565c6d4.png)